.env

```
@@ -2,22 +2,22 @@
 VERBOSE_HARNESS=0
 
 # Used to determine sandbox build type:
-TYPE="channel" # OR "source"
+TYPE="source" # OR "source"
 
 # Used when TYPE==channel:
 ALGOD_CHANNEL="nightly"
 
 # Used when TYPE==source:
 ALGOD_URL="https://github.com/algorand/go-algorand"
-ALGOD_BRANCH="master"
+ALGOD_BRANCH="feature/avm-box"
```

TODO: Change back to `master`

---

After `feature/box-storage` is merged in go-algorand:
```suggestion
TYPE="channel" # OR "source"
```

---

After `feature/box-storage` is merged in go-algorand:
```suggestion
ALGOD_CHANNEL="master"
```